contrib/python-bindings/include/triangulation_wrapper.h

```diff
@@ -137,6 +145,20 @@ namespace python
       boost::python::list &material_id,
       const bool           colorize = false);
 
+    /**
+     * This function produces a square in the xy-plane with a cylindrical
+     * hole in the middle. The square and the circle are centered at the
+     * origin. In 3d, this geometry is extruded in z direction to the
+     * interval [0,L].
```

Comment:
Is there a way to use `@copydoc` to just copy the documentation of the function I assume you're calling internally?

Reply:
Like that?

Reply:
If it works here, should I replace it everywhere (in a separate PR)? It is rather annoying to copy-paste...

Reply:
Yes, I think that would be great -- this way things don't go out of date.

Have you checked that it actually works?

(I wished we could do the same for the `_docstring` text...

Reply:
> Have you checked that it actually works?

Hm, how do I do that? The documentation for the python bindings is not compiled as far as I can see...

Reply:
Oh, I see. That's unfortunate. Let's just assume that it works, then.